exla/lib/exla.ex

```diff
@@ -220,7 +220,7 @@ defmodule EXLA do
 
```

Comment:
I think those are unrelated, we just want to pass fixed integers as XLA inputs. We could actually make those unsigned (and change to uint on the c++ side), since all of those are non-negative sizes. Doesn't matter much, since those are just a few values, your call!

Reply:
I opened a PR to change to u64 https://github.com/elixir-nx/nx/pull/1526.